tests/test_packagemanager.py

```diff
@@ -0,0 +1,84 @@
+import json
+import mock
+import requests
+
+from dcos import packagemanager
+
+
+describe_response_content_type = 'application/vnd.dcos.package.describe-response+json;charset=utf-8;version=v2'
```

Comment:
and then further revised this in e499aeb and I think the result is a lot nicer, because I can just use the `packagemanager` and not poke into `dcos.cosmos` at all.